spec/controllers/status_messages_controller_spec.rb

```diff
@@ -158,6 +158,13 @@
 #      response.body.should include('Status message requires a message or at least one photo')
 #    end
 
+    it 'has one participation' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.